src/python/pants/notes/2.9.x.md

```diff
@@ -1,5 +1,39 @@
 # 2.9.x Release Series
 
+## 2.9.0.dev2 (Nov 18, 2021)
+
+### New Features
+
+* [jvm] Add support for compiling cycles between Java and Scala ([#13653](https://github.com/pantsbuild/pants/pull/13653))
```

Comment:
Are we documenting JVM in 2.9 now?

Reply:
IMO, yes. We're starting to blog about it, so it's time.